R/tab_create_modify.R

```diff
@@ -632,7 +632,25 @@ set_footnote <- function(loc, data, footnote) {
 
 set_footnote.cells_title <- function(loc, data, footnote) {
 
-  if ((loc$groups %>% rlang::eval_tidy()) == "title") {
+  title_components <- rlang::eval_tidy(loc$groups)
+
+  if (
+    length(title_components) == 2 &&
+    all(title_components %in% c("title", "subtitle"))
+  ) {
+
+    data <-
+      dt_styles_add(
+        data = data,
+        locname = c("title", "subtitle"),
+        grpname = NA_character_,
+        colname = NA_character_,
+        locnum = c(1, 2),
+        rownum = NA_integer_,
+        styles = footnote
+      )
+
+  } else if (title_components == "title") {
```

Comment:
Instead of if/elseif/elseif, I would make it two completely separate conditionals (one for title, one for subtitle) that are not mutually exclusive.